tests/unit/modules/dcnm/fixtures/dcnm_network.json

```diff
@@ -8,6 +8,46 @@
         "10.10.10.227": "XYZKSJHSMK4",
         "10.10.10.228": "XYZKSJHSMK5"
     },
+    "fabric_associations": {
+        "RETURN_CODE": 200,
+        "METHOD": "GET",
+        "REQUEST_PATH": "https://mock-ndfc/appcenter/cisco/ndfc/api/v1/lan-fabric/rest/control/fabrics/msd/fabric-associations",
+        "MESSAGE": "OK",
+        "DATA": [
+            {
+                "fabricId": 1,
+                "fabricName": "test_network",
```

Comment:
Just a minor thing but can we change the name of this fabric to be "standalone_fabric" instead of "test_network".  I know we have the other info below that makes it clear it's a standalone fabric but just visually easier to read when debugging issues in the future.